src/components/taskDetails/index.tsx

```diff
@@ -63,8 +64,8 @@ type Props = {
 const TaskDetails: FC<Props> = ({ taskID }) => {
     const router = useRouter();
     const { isUserAuthorized } = useUserData();
-    const [newEndOnDate, setNewEndOnDate] = useState('');
     const [isEditing, setIsEditing] = useState<boolean>(false);
+    const [loading, setLoading] = useState<boolean>(false);
```

Comment:
Do we need this as RTK query give loading state?

Reply:
No I tested, it does not show loading true when I save it, so for button I needed it separately, Please let me know if any misunderstanding.